simulation_run.py

```diff
@@ -51,13 +52,14 @@ def main():
     scenario_params_list = config["scenario_params_list"]
     n_repeats = config["n_repeats"]
 
-    # GPU config
-    #gpus = tf.config.experimental.list_physical_devices("GPU")
-    #tf.config.experimental.set_memory_growth(gpus[0], True)
+    # # GPU config
```

Comment:
One extra `#` here

Reply:
I will clean these 2 lines soon !

Reply:
What is the right way to comment a bunch of lines of which one of them is a comment? I did it that way so that the whole bunch can be uncommented, without the actual comment to be uncommented. Is this a wrong way to do it @natct10?

Reply:
Yep, no problem!